setup.cfg

```diff
@@ -1,24 +1,13 @@
 [flake8]
-max-line-length = 120
+max-line-length = 88
 extend-ignore = E203, W503
 
 [mypy]
 python_version = 3.6
```

Comment:
Should we bump this as well?

Reply:
I think it can be bumped to 3.7 in next PR that removes dependency on dateutil (Python 3.7 ships `datetime.fromisoformat`)